cmd/immudb/command/init.go

```diff
@@ -84,6 +84,8 @@ func (cl *Commandline) setupFlags(cmd *cobra.Command, options *server.Options) {
 	cmd.Flags().Duration("session-timeout", 2*time.Minute, "session timeout is a duration after which an inactive session is forcibly closed by the server")
 	cmd.Flags().Duration("sessions-guard-check-interval", 1*time.Minute, "sessions guard check interval")
 	cmd.Flags().MarkHidden("sessions-guard-check-interval")
+	cmd.Flags().Bool("reflection", options.ReflectionServerEnabled, "GRPC relection server enabled")
```

Comment:
typo `relection`

`reflection` what's about?

Reply:
Allows to use tools like
https://github.com/fullstorydev/grpcui
Or
https://blog.postman.com/latest-advancements-to-postmans-grpc-support/

to automatically reflect database methods in interactive clients

Reply:
Resolved in https://github.com/codenotary/immudb/pull/1546